release-notes.md

```diff
@@ -1,5 +1,14 @@
 # Release notes
 
+## July 9, 2019
+
+The following updates were made in this release:
+
+**iOS Analytics 2.1.2**
+
+- ACPAnalytics now correctly identifies Acquisition link event types
+- Fixes a compile time error when using the “-all_load” linker flag
```

Comment:
- please add a period at the end of each bullet point.
- Please add a hyphen between "compile" and "time".